pkg/k8s/k8s.go

```diff
@@ -62,7 +64,9 @@ func WaitForDeploymentDeletion(cs *clients.Clients, namespace, name string) erro
 		log.Printf("Waiting for deletion of %s deployment\n", name)
 		return false, nil
 	})
-	assert.NoError(err, fmt.Sprintf("%s Deployment deletion failed\n", name))
+	if err != nil {
+		testsuit.T.Errorf("failed to delete deployment %s \n %v", name, err)
```

Comment:
was `\` added to show error in the next line?

Reply:
Yes, we were already using that kind of formatting so I started using it everywhere.